nette/notejam/app/presenters/templates/Note/delete.latte

```diff
@@ -0,0 +1,9 @@
+{block title}{$note->name}{/block}
+{block header}{$note->name}{/block}
+{block content}
+	<p>Are you sure you want to delete the note?</p>
+	{form deleteNoteForm}
```

Comment:
It would be nice to keep these form macros to show possibilities. And a lot of other macros are still in use in other places and I don't think the editors understand them somehow better :)